src/backend/commands/trigger.c

```diff
@@ -70,6 +70,7 @@ int			SessionReplicationRole = SESSION_REPLICATION_ROLE_ORIGIN;
 
 /* How many levels deep into trigger execution are we? */
 static int	MyTriggerDepth = 0;
+List *triggerOids = NIL; /* To store all trigger calls information */
```

Comment:
If this variable is only used in extension, why not put it in the extension side ?

Reply:
I missed setting this variable in previous commit. It is set in trigger.c as 
 triggerOids = lappend_oid(triggerOids, trigdata->tg_trigger->tgoid);